### ISSUE TITLE
Can't have EFS controller and EBS node on the same workers

### ISSUE THREAD

**Issue by bodgit** [kubernetes-sigs/aws-efs-csi-driver#445]
/kind bug

**What happened?**

I have a new cluster with 3 worker nodes and I've tried to deploy both the EBS storage driver and this EFS storage driver to the cluster. I've just noticed that I end up with either two of the EFS controller, or the EBS node Pods stuck in Pending state:
```
  Warning  FailedScheduling   56s (x28 over 32m)     default-scheduler   0/3 nodes are available: 3 node(s) didn't have free ports for the requested pod ports.
```
Looking through the manifests, the ports can be summarised as follows:
| Pod | Port | hostNetwork |
| --- | ---- | ----------- |
| EBS node | 9808 | true |
| EBS controller | 9808 | false |
| EFS node | 9809 | true |
| EFS controller | 9808 | true |

You can see both the EBS node and EFS controller both use the same port and have `hostNetwork: true` which I think means they'll never exist on the same worker node which is a problem as the EBS node will try and run everywhere due to being a DaemonSet.

**What you expected to happen?**

Both storage drivers should coexist.

**How to reproduce it (as minimally and precisely as possible)?**

Deploy both storage drivers to the same cluster. You'll end up with either two EFS controller or EBS node Pods in Pending state.

**Anything else we need to know?**:

**Environment**
- Kubernetes version (use `kubectl version`): 1.17
- Driver version: v1.2.0

Is the fix simply to remove the `hostNetwork: true` from the EFS controller manifest given the EBS controller doesn't have that?

**Comment by bodgit** [comment]
This seems to have only appeared after v1.2.0 was released and I had tried using v1.2.1 because I saw the tag. Going back to v1.2.0 fixes this for now, but I wonder if that `hostNetwork: true` should really be there anyway?

**Comment by wongma7** [comment]
yes it is intentional.

are you using helm or kustomize to install?

This should have been fixed by helm chart 1.2.4 but let me check if the kustomize template was als ofixed.

**Comment by bodgit** [comment]
I was using kustomize to install.
